docs/weave-tutorial/10-batch-tx.md

```diff
@@ -0,0 +1,60 @@
+---
+id: batch-txs
+title: Batch Transactions
+sidebar_label: Batch Transactions
+---
+
+I don't think you will not have patience to wait for posting thousands of transactions one by one right?
+
+Via wrapping vanilla `Messages` into a `repeated` **Union** object that includes variety of messages, we can achieve the ease of use of batch transactions; thus, this will prevent the blockhain from being flooded by thousands of messages.
```

Comment:
```suggestion
Via wrapping vanilla `Messages` into a `repeated` **Union** object that includes variety of messages, we can achieve the ease-of-use of batch transactions; this will prevent the blockhain from being flooded by thousands of messages.
```